intensity/temp_extract.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#PBS -P w85
+#PBS -qnormal
+#PBS -N temp_extract
+#PBS -m abe
+#PBS -M kieran.ricardo@ga.gov.au
+#PBS -l walltime=1:30:00
+#PBS -l mem=96GB,ncpus=48,jobfs=4000MB
+#PBS -W umask=0002
+#PBS -joe
+#PBS -l storage=gdata/v10+gdata/gb6
+
+module purge
+
+module use /g/data/v10/public/modules/modulefiles
+module load dea/20210527
```

Comment:
is this interchangeable with 

```
module use /g/data/hh5/public/modules
module load conda/analysis3
```

Reply:
I'm not in hh5 but I only us:

- xarray
- gdal
- numpy
- pyproj
- scipy
- mpi4py